curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/nest-many-elements-within-a-single-div-element.md

```diff
@@ -17,9 +17,9 @@ Just like any other non-self-closing element, you can open a `div` element with
 
 # --instructions--
 
-Nest your "Things cats love" and "Things cats hate" lists all within a single `div` element.
+Nest your "`Things cats love`" and "`Things cats hate`" lists all within a single `div` element.
```

Comment:
```suggestion
Nest your "Things cats love" and "Top 3 things cats hate" lists all within a single `div` element.
```
This is kinda awkward, since these are just references to lists. Again I think it's best to leave it to translators to work out how best to refer to something that's written in English in code.

I also changed the second list 'title' to match the seed.